.storybook/themes/news.scss

```diff
@@ -95,8 +95,8 @@
 		"spacing-16": 7rem,
 		// 112px
 		"spacing-17": 7.5rem,
-
 		// 120px
+		//Font Weight
```

Comment:
weird is this new?

Reply:
Yeah I added it, there was no comment for it. It was also unclear that the comment about the value was below the value, so adding this helped with formatting

Reply:
this is good. helps to read